bitcoin/src/address.rs

```diff
@@ -848,10 +879,20 @@ impl Address {
 	self.address_type_internal()
     }
 
+    /// Checks whether or not the address is following Bitcoin standardness rules when
+    /// *spending* from this address. This method should *NOT* called by senders. For forward
+    /// compatibility, the senders must send to any [`Address`]. Receivers can use this method to
+    /// check whether or not they can spend from this address.
```

Comment:
I like having a warning in summary but not sure if this long is appropriate. Maybe something like "Checks whether the address obeys Bitcoin standardness rules when *spending* from this address. **NOT* to be called by senders!" And full explanation in description. Anyway, I won't block the PR because of it.

Reply:
Changed